### PR TITLE
Adding a french translation

### PR DESCRIPTION
**Pull request by RyDroid**
https://github.com/HassanHeydariNasab/defendo/issues/5#issuecomment-329991425

### CODE REVIEW

File: Cxenoj.csv [hunk 1]
```diff
@@ -1,35 +1,41 @@
-id,en
+id,en,fr
 "Saluton homido,
 Ni bezonas komandanton por defendi la bazon kontraux malamikoj.
 Se antauxe vi helpis nin, mi ne memoras; Cxar nun estas iam antaux tiam!","Hello humankind,
 We need a commander to defend the base against enemies.
-If already you helped us, I don't remember; Because now is a time before that!"
```
Comment (HassanHeydariNasab) [inline Cxenoj.csv]: Sorry, it's obsoleted. It was about a humor with time!

File: Cxenoj.csv [hunk 2]
```diff
@@ -1,35 +1,41 @@
...
-Ludi,Play
-Cxu vere vi volas eliri?,Do you really want to exit?
+Shooting by laser and ondilo costs 5 dollars and shooting by cannon costs 1 dollar.","Sais tu que tu peux améliorer les armes ? Collecte des armes similaires dans la base oy met une arme dans la base et achète la même arme.
+Fait un clic droit ou un appui long sur un canon ou un laser pour changer sa stratégie.
+Les armes ont besoin d'énergie pour tirer sur les ennemis; met les sur les cercles. (Tu peux aussi améliorer les bombes de cette manière.)
+Tirer par laser ou ondilo coûte 5 et tirer par canon coûte 1."
+Mi estas preta.,I'm ready.,Je suis prêt·e.
+Ondo,Wave,Vague
+Nova,New,Nouveau
+MalNova,Old,Ancien
```
Comment (HassanHeydariNasab) [inline Cxenoj.csv]: is it similar to "Ancient"? it's not too old.

File: ikono.svg [hunk 3]
```diff
@@ -1,6 +1,4 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!-- Created with Inkscape (http://www.inkscape.org/) -->
```
Comment (HassanHeydariNasab) [inline ikono.svg]: why?

File: ikono.svg [hunk 4]
```diff
@@ -12,7 +10,6 @@
    width="512"
    height="512"
    viewBox="0 0 512.00001 512.00001"
-   id="svg2"
```
Comment (HassanHeydariNasab) [inline ikono.svg]: why?

File: Cxenoj.csv [hunk 5]
```diff
@@ -1,35 +1,41 @@
...
-Vi Venkigxis!,You Win!
-Vi Malvenkigxis!,You Lose!
-Ondoj,Waves
-Eliri,Exit
-Ludi,Play
-Cxu vere vi volas eliri?,Do you really want to exit?
+Shooting by laser and ondilo costs 5 dollars and shooting by cannon costs 1 dollar.","Sais tu que tu peux améliorer les armes ? Collecte des armes similaires dans la base oy met une arme dans la base et achète la même arme.
+Fait un clic droit ou un appui long sur un canon ou un laser pour changer sa stratégie.
+Les armes ont besoin d'énergie pour tirer sur les ennemis; met les sur les cercles. (Tu peux aussi améliorer les bombes de cette manière.)
+Tirer par laser ou ondilo coûte 5 et tirer par canon coûte 1."
```
Comment (HassanHeydariNasab) [inline Cxenoj.csv]: Dollar makes a natural feel

File: ikono.svg (hunk 3 above)
Comment (RyDroid) [reply ikono.svg]: It is useless and will never be useful, so I see no valid to keep this metadata.

File: ikono.svg (hunk 4 above)
Comment (RyDroid) [reply ikono.svg]: It is useless and will never be useful, so I see no valid to keep this identifier.

File: Cxenoj.csv (hunk 5 above)
Comment (RyDroid) [reply Cxenoj.csv]: To me, it does not feel natural, it sounds USA. I did most of life in France, that probably explains my feeling. Do you want that I add "dollar"?

File: Cxenoj.csv (hunk 2 above)
Comment (RyDroid) [reply Cxenoj.csv]: It is not as precise as that. In French, there are other words to indicate that something is very old.
https://fr.wiktionary.org/wiki/ancient#Adjectif
https://www.linguee.com/english-french/search?source=auto&query=ancient
https://www.linguee.com/english-french/search?source=auto&query=old

File: ikono.svg (hunk 3 above)
Comment (HassanHeydariNasab) [reply ikono.svg]: When I save the file by inkscape, it puts its metadata to the file. I prefer to keep it with respect to inkscape.

File: ikono.svg (hunk 4 above)
Comment (HassanHeydariNasab) [reply ikono.svg]: I prefer to keep it :)

Comment (RyDroid) [reply ikono.svg]: Ok.

File: ikono.svg (hunk 3 above)
Comment (RyDroid) [reply ikono.svg]: Ok.

**Pull request merged**